src/components/structures/ToastContainer.js

```diff
@@ -74,7 +74,7 @@ export default class ToastContainer extends React.Component {
         });
 
         return (
-            <div className={containerClasses}>
+            <div className={containerClasses} role="alert" aria-live="polite">
```

Comment:
the alert role must be present even if `totalCount === 0`

Reply:
things inside an an aria live region will only be spoken if they are added since the live region is created, not if they exist when the live region is created